io_scene_godot/converters/simple_nodes.py

```diff
@@ -17,6 +17,8 @@ def export_empty_node(escn_file, export_settings, node, parent_gd_node):
         return parent_gd_node
     empty_node = NodeTemplate(node.name, "Spatial", parent_gd_node)
     empty_node['transform'] = node.matrix_local
+    print(node.matrix_local)
```

Comment:
remove these prints before merging

Reply:
thanks for reviewing!I have no much experience on Github, so which way is the best to remove there prints?

Reply:
2 commits has pushed.

Reply:
Looks like you figured it out 👍